src/Imagick.php

```diff
@@ -1450,7 +1454,16 @@ public function getImageColors()
     /** @return int */
     public function getImageColorspace()
     {
-        throw new Exception(sprintf('%s::%s not implemented', __CLASS__, __FUNCTION__));
+        $files = $this->files;
+        $current = key($files);
+        $files[$current][] = new Argument('format', '%[colorspace]');
+        $files[$current][] = 'info:';
+
+        $convert_command = $this->buildConvertCommand($files, '-');
+        $colorspace = shell_exec($convert_command);
+
+        $name = 'Imagick::COLORSPACE_'.strtoupper($colorspace);
```

Comment:
Is this intended to be used as a constant elsewhere, or just to be descriptive?

Reply:
Not sure if I understand, [`getImageColorspace`](https://www.php.net/manual/en/imagick.getimagecolorspace.php) returns an Int and you already provided the constants (I just looked how you did it in another method :D)